package/Dockerfile

```diff
@@ -113,10 +113,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl /usr/bin/channelserver && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.5.6-rc6
-ENV CATTLE_DASHBOARD_UI_VERSION v2.5.6-rc6
-ENV CATTLE_CLI_VERSION v2.4.11-rc2
-
+ENV CATTLE_UI_VERSION 2.5.6
+ENV CATTLE_DASHBOARD_UI_VERSION v2.5.6
+ENV CATTLE_CLI_VERSION v2.4.10
```

Comment:
why isn't this v2.4.11?

Reply:
`v2.4.11` had no new changes, so we didn't bump the version